### PR TITLE
Help returns exit 0 code

### LINKED ISSUE #313: `sda-cli help` or `sda-cli -h` etc, should return `exit 0`
**Describe the bug**
See title

**Steps to reproduce**
```
$ ./sda-cli help
...
$ echo $?
1
```
**Expected behavior**
- [ ] exit status of `/sda-cli help` (or `-h` etc) should be 0

**Estimation of size**: small

### PR DESCRIPTION
**Pull request by pahatz**
**Related issue(s) and PR(s)**  
This PR closes #313 

**Description**
Currently sda-cli does not return the correct exit code when we run help or -h (exit 1)
It should return exit code 0 except the case of asking for a wrong subcommand.

**How to test**
Run:
```
./sda-cli help
./sda-cli -h
./sda-cli help list
./sda-cli help inexistent_command
```
followed each time by:
```
echo $?
```
to get the exit code of the previous command.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -86,17 +86,16 @@ func main() {
 // of the arguments for the subcommand.
 func ParseArgs() (string, []string) {
 	// Print usage if no arguments are provided.
-	// Terminate with non-zero exit status.
 	if len(os.Args) < 2 {
 		_ = Help("help")
-		os.Exit(1)
+		os.Exit(0)
```
Comment (pontus) [inline main.go:91]:
```
pontus@lalage:~/Utveckling/sda-cli$ ./sda-cli illegal_command 2>/dev/null ; echo $?
0
pontus@lalage:~/Utveckling/sda-cli$ 
```

is that intentional?

Comment (pahatz) [reply main.go:91]: Right, that should be 1.

File: main.go [hunk 2]
```diff
@@ -138,7 +138,7 @@ func ParseArgs() (string, []string) {
 	// non-zero exit status.
 	if len(os.Args) == 1 {
 		_ = Help(command)
-		os.Exit(1)
+		os.Exit(0)
```
Comment (MalinAhlberg) [inline main.go:141]: exit 1 seems correct here? For example, the command `sda-cli createKey` lacks the required `<name>` and is invalid.

**Review by MalinAhlberg — COMMENTED** [review]
Well done, and thanks for the nice instructions! I think there is one more case where exit 1 should be returned, made a comment. Apart from that, I'm ready to approve.

**Review by MalinAhlberg — APPROVED** [review]

**Review by jbygdell — APPROVED** [review]

**Pull request merged**